### PR TITLE
Add 2 scenarios for vars when installroot is used

### PR DESCRIPTION
**Pull request by j-mracek**

### CODE REVIEW

File: dnf-behave-tests/features/vars-releasever.feature [hunk 1]
```diff
@@ -86,3 +86,63 @@ Scenario: Releasever is substituted in baseurl via a value detected from 'system
     And Transaction is following
         | Action        | Package                       |
         | install       | setup-0:2.12.1-1.fc29.noarch  |
+
+@destructive
+Scenario: Releasever is substituted in baseurl using vars loaded from location (host or installroot) like used repos
```
Comment (m-blaha) [inline dnf-behave-tests/features/vars-releasever.feature]: The scenario name doesn't sound right to me. What about
Releasever is substituted in baseurl using vars loaded from the same location (host or installroot) as repos

File: dnf-behave-tests/features/vars-releasever.feature [hunk 2]
```diff
@@ -86,3 +86,63 @@ Scenario: Releasever is substituted in baseurl via a value detected from 'system
...
+        enabled=1
+        gpgcheck=0
+        """
+
+
+    And I create and substitute file "/etc/dnf/vars/releasever" with
+        """
+        0123
+        """
+    And I do not set reposdir
```
Comment (m-blaha) [inline dnf-behave-tests/features/vars-releasever.feature]: this is redundant. You've already executed this step on line 93

File: dnf-behave-tests/features/vars-releasever.feature [hunk 3]
```diff
@@ -86,3 +86,63 @@ Scenario: Releasever is substituted in baseurl via a value detected from 'system
     And Transaction is following
         | Action        | Package                       |
         | install       | setup-0:2.12.1-1.fc29.noarch  |
+
+@destructive
+Scenario: Releasever is substituted in baseurl using vars loaded from location (host or installroot) like used repos
+  Given I do not set releasever
+    And I do not set reposdir
+    And I copy directory "{context.dnf.repos_location}/dnf-ci-fedora" to "/temp-repos/base-f0123"
+    And I create and substitute file "/etc/yum.repos.d/test.repo" with
```
Comment (m-blaha) [inline dnf-behave-tests/features/vars-releasever.feature]: you can create repo file right in the final destination by using leading double slash (that means the path will not be prepended by installroot):
```
And I create and substitute file "//etc/yum.repos.d/test.repo" with
```
Than the moving part on lines 111-112 is obsolete.

**Comment by m-blaha** [comment]
Sorry, due to https://github.com/rpm-software-management/ci-dnf-stack/pull/647, you will need to rebase the PR.

**Comment by j-mracek** [comment]
@m-blaha Rebased, refactored

**Pull request merged**